sites/public/src/components/listing/ListingViewSeedsHelpers.tsx

```diff
@@ -119,29 +119,34 @@ export const getHasNonReferralMethods = (listing: Listing) => {
 }
 
 export const getAccessibilityFeatures = (listing: Listing) => {
-  let featuresExist = false
-  const features = Object.keys(listing?.listingFeatures ?? {}).map((feature, index) => {
-    if (listing?.listingFeatures[feature]) {
-      featuresExist = true
+  const enabledFeatures = Object.entries(listing?.listingFeatures ?? {})
```

Comment:
I refactored these two methods because there was a bug with commas sometimes showing up at the end of a list. This filters out only the values which are true first, then loops through them.